internal/pkg/model/state.go

```diff
@@ -53,16 +53,28 @@ func (s *State) ReloadPathsState() error {
 
 	// Track all known paths
 	for _, object := range s.All() {
-		if object.Manifest().State().IsPersisted() {
-			s.TrackRecord(object.Manifest())
-		}
+		s.TrackRecord(object.Manifest())
 	}
 	return nil
 }
 
 func (s *State) TrackRecord(record Record) {
-	for _, path := range record.GetRelatedPaths() {
-		s.pathsState.MarkTracked(path)
+	if !record.State().IsPersisted() {
```

Comment:
4: ak je `Record` `invalid = true`, ... tak sme nenacitali vsetky jeho subory.
Priklad, subor `meta.json` je nevalidny, .... tak uz ani neskusame nacitat `config.json` a dalsie subory.

No a tu bol problem, ze tieto dalsie subory chybali medzi `tracked paths`, ... a namiesto 1 chyby, sa vypisali 2.
A to aj to, ze su nejake `untracked files`, pritom tie subory tam byt maju.

Riesenim je, ze ak je subor `invalid = true`, tak sa oznacia vsetky jeho pod-subory ako `tracked`, ... user musi fixnut najprv primarnu chybu, a az potom mozeme riesit, ci tam nie je nejaky neocakavany subor.